tests/trestle/core/commands/add_test.py

```diff
@@ -111,13 +113,17 @@ def test_run_failure():
     """Test failure of _run for AddCmd."""
     testargs = ['trestle', 'add', '-e', 'catalog.metadata.roles']
     with patch.object(sys, 'argv', testargs):
-        with pytest.raises(SystemExit):
+        with pytest.raises(SystemExit) as e:
             Trestle().run()
+        assert e.type == SystemExit
+        assert e.value.code == 2
 
     testargs = ['trestle', 'add', '-f', './catalog.json']
     with patch.object(sys, 'argv', testargs):
-        with pytest.raises(SystemExit):
+        with pytest.raises(SystemExit) as e:
             Trestle().run()
+        assert e.type == SystemExit
+        assert e.value.code == 2
 
```

Comment:
Just want to check but looks like the argparse method will automatically throw a sys exit (inside of code we have not written)?